ansible-playbook/roles/gophish/tasks/main.yml

```diff
@@ -34,11 +39,45 @@
   apt:
     upgrade: safe
 
+- name: Ensure /etc/ssl/csr folder exists
+  file:
+    path: /etc/ssl/csr
+    state: directory
+    mode: '0755'
+  
+- name: Ensure /etc/ssl/private folder exists
+  file:
+    path: /etc/ssl/private
+    state: directory
+    mode: '0755'
+
+- name: Ensure /etc/ssl/crt folder exists
+  file:
+    path: /etc/ssl/crt
+    state: directory
+    mode: '0755'
+
+- name: Generate an OpenSSL private key with the default values (4096 bits, RSA)
+  openssl_privatekey:
```

Comment:
It looks like the [`openssl_*`](https://docs.ansible.com/ansible/latest/modules/openssl_privatekey_module.html) modules require either the `cryptography` or `pyOpenSSL` Python packages to work. When doing this on a vanilla Debian system, I got an error since neither of these were installed.

It seems that we could add `python-pip` to the `{{install_packages}}`, bump up the "Install specified packages." task up before the cert generation tasks, and add a task that installs the `cryptography` Python module.

Something like:

```yml
- name: Ensure the cryptography Python package is installed
  pip:
    name: cryptography
```

Reply:
I added it!